src/CodeEditor.ts

```diff
@@ -0,0 +1,64 @@
+import { EditorState, basicSetup } from "@codemirror/basic-setup";
+import { EditorView, keymap } from "@codemirror/view";
+import { indentWithTab } from "@codemirror/commands";
+import { javascript } from "@codemirror/lang-javascript";
+import { ProgrammingLanguage } from "./ProgrammingLanguage";
+import { python } from "@codemirror/lang-python";
+import { LanguageSupport } from "@codemirror/language";
+// import { autocompletion } from "@codemirror/autocomplete";
```

Comment:
Why is this commented?

Reply:
I intend to use it when resolving https://github.com/dodona-edu/papyros/issues/46, so this is just to remove the unused identifier warning. Removing it would achieve the same goal but I intend to work on that issue in the near future.